tests/contracts/sources.ts

```diff
@@ -285,4 +285,74 @@ export const contractSources: { [key: string]: string } = {
             chainid = block.chainid;
         }
     }`,
+  JoinCandidatesWrapper: `
+    pragma solidity >=0.8.0;
+
+    interface ParachainStaking {
+        // First some simple accessors
```

Comment:
Soon, we'll use this interface in more tests. Is there a good way to reuse that code in these tests?

Reply:
No I don't think it's possible to share dependancies across source codes with the way we're managing contracts currently

Reply:
This would be solved by https://github.com/PureStake/moonbeam/pull/383

Reply:
sweet :)